D:SWARM-Use-Case-at-SLUB-Dresden.md

```diff
@@ -0,0 +1,59 @@
+At [SLUB Dresden](http://www.slub-dresden.de), the *Streaming version* of d:swarm is used for normalising data retrieved from various sources before feeding it to a [Solr](http://lucene.apache.org/solr/) server. Clients such as the new [SLUB Katalog (beta)](http://katalogbeta.slub-dresden.de/) can then request data from Solr and present it to library users.
```

Comment:
"... feeding it to a Solr server" -> "... feeding it into a Solr search index" (?)